mmdeploy/backend/torchscript/wrapper.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+from typing import Dict, Optional, Sequence, Union
+
+import torch
+
+from mmdeploy.utils import Backend
+from mmdeploy.utils.timer import TimeCounter
+from ..base import BACKEND_WRAPPER, BaseWrapper
+from .init_plugins import get_ops_path
+
+
+@BACKEND_WRAPPER.register_module(Backend.TORCHSCRIPT.value)
+class TorchscriptWrapper(BaseWrapper):
+    """Torchscript engine wrapper for inference.
+
+    Args:
+        model (torch.jit.RecursiveScriptModule): torchscript engine to wrap.
+        input_names (Sequence[str] | None): Names of model inputs  in order.
```

Comment:
`input_names` and `output_names` have no default value as shown in line 39 and line 40